doc/changes/changes_5.0.0.md

```diff
@@ -14,9 +14,12 @@ This release improves verification of Maven projects with multiple modules. `pro
 
 The release also also updates the Eclipse formatter settings to never join wrapped lines and to indent `case` inside of `switch`. This helps avoid forcing line wraps with trailing `//` comments.
 
+The release also runs the CI build workflow also when a PR is converted from "draft" to "ready for review". This ensures that the complete build runs even if some workflow steps are skipped for draft PRs.
+
```

Comment:
The explanation in the linked ticket #620 is excellent. 
Would it make sense to copy the explanation to either developer guide or at least the changelog?